controllers/limited_support/limited_support_controller.go

```diff
@@ -34,6 +35,8 @@ const (
 	limitedSupportConfigMapNamespace = "openshift-osd-metrics"
 )
 
+const EnvClusterID = "CLUSTER_ID"
```

Comment:
Since this matches https://github.com/openshift/osd-metrics-exporter/blob/d834f20ed0a2314a0bda8b8591dc2b20df1f6251/controllers/proxy/proxy_controller.go#L33

Should we just define the consts in one place? Maybe `controllers/utils/const.go`? Then the proxy controller and this limited support controller could just use `utils.EnvClusterID` and reference the same constant.

Reply:
Makes sense, thanks